contrib/externs/jquery-1.9.js

```diff
@@ -645,11 +651,20 @@ jQuery.Event.prototype.isImmediatePropagationStopped = function() {};
 jQuery.Event.prototype.isPropagationStopped = function() {};
 
 /** @type {boolean} */
+jQuery.Event.prototype.isTrigger;
```

Comment:
I don't believe this is a documented property of jQuery events. We only provide externs for documented public properties.